jerry-core/parser/js/js-scanner-util.c

```diff
@@ -1819,6 +1816,8 @@ scanner_create_variables (parser_context_t *context_p, /**< context */
         scope_stack_reg_top++;
       }
       continue;
+    } else {
+      JERRY_ASSERT (context_p->scope_stack_size != 0);
```

Comment:
There is a `continue` before it so no need `else`

Reply:
Indeed, I've updated the patch